src/index.html

```diff
@@ -8,8 +8,25 @@
 </head>
 <body>
     <h1>Leaderboard</h1>
-    <h1>Leaderboard</h1>
-    <h1>Leaderboard</h1>
+    <div class="container">
+        <div class="list">
+            <div class="list-title">
+                <h2>Recent Scores</h2>
+                <input type="submit" name="add" id="add" value="Refresh" />
+            </div>
+            <table class="recent-scores"></table>
+        </div>
+        <div class="input-form">
+            <h2>Add Your Score</h2>
+            <form action="#" class="add-score">
+                <input type="text" name="name" id="name" placeholder="Your Name" />
+                <input type="text" name="score" id="score" placeholder="Your Score" />
```

Comment:
- [ ] [OPTIONAL] It could be better to change your `score` input in `type="number"`, you'll have something like `<input type="number" name="score" id="score" placeholder="Your Score" required/>`. Kindly fix it 😃